src/Nethermind/Nethermind.Db/StandardDbInitializer.cs

```diff
@@ -12,33 +12,30 @@ namespace Nethermind.Db
     public class StandardDbInitializer : RocksDbInitializer
     {
         private readonly IFileSystem _fileSystem;
-        private readonly bool _fullPruning;
 
         public StandardDbInitializer(
             IDbProvider? dbProvider,
             IRocksDbFactory? rocksDbFactory,
             IMemDbFactory? memDbFactory,
-            IFileSystem? fileSystem = null,
-            bool fullPruning = false)
+            IFileSystem? fileSystem = null)
             : base(dbProvider, rocksDbFactory, memDbFactory)
         {
             _fileSystem = fileSystem ?? new FileSystem();
-            _fullPruning = fullPruning;
         }
 
-        public void InitStandardDbs(bool useReceiptsDb)
+        public void InitStandardDbs(bool useReceiptsDb, bool useBlobDb = true)
```

Comment:
Cosmetic, we have receipt**s**, header**s**. Shouldn't we have useBlob**s**Db?

Reply:
Right, I will correct it